src/Installers/Windows/AspNetCoreModule-Setup/Forwarders/build.proj

```diff
@@ -0,0 +1,24 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="15.0" DefaultTargets="Build">
+  <PropertyGroup Label="Globals">
+    <VCProjectVersion>17.0</VCProjectVersion>
+    <RootNamespace>aspnetcorev2</RootNamespace>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
+    <PlatformToolset>v143</PlatformToolset>
```

Comment:
nit: remove, this should be set by https://github.com/dotnet/aspnetcore/blob/main/eng/targets/Cpp.Common.props

Reply:
Resolved.